### ISSUE TITLE
Python 3.8->3.9  upgrade error   -  no attribute 'isAlive'. Did you mean: 'is_alive'?

### ISSUE THREAD

**Issue by Pcrossley69** [kamailio/kamcli#24]
Installed Kamcli on Jammy using package manager and kamailio repository

When using any jsonrpc command I get a stack backtrace that ends as follows:

 File "/usr/lib/python3/dist-packages/kamcli/iorpc.py", line 255, in command_jsonrpc_fifo
    if not tiofifo.isAlive():
AttributeError: 'IOFifoThread' object has no attribute 'isAlive'. Did you mean: 'is_alive'?

Apparently this is because Python 3.9  replaced  isAlive() with is_alive()

**Comment by Pcrossley69** [comment]
I edited /usr/lib/python3/dist-packages/kamcli/iorpc.py   to replace isAlive with is_alive
but this then generated another error:

AttributeError: 'str' object has no attribute 'decode'. Did you mean: 'encode'?

When researching this I found an explanation:
"Python 3 stores a string as a sequence of Unicode code points and hence there's no need to decode them.
The new str class doesn't even implement the decode()"

I eventually got it working by running this script 
I've only edited the iorpc.py file - I'm not sure whether this finds all the decodes that need removing 
but it worked for the few commands that I tested.

ed /usr/lib/python3/dist-packages/kamcli/iorpc.py <<"EOT"
1,$ s/isAlive/is_alive/
1,$ s/response.decode()/response/
w
q
EOT

**Comment by Pcrossley69** [comment]
Unfortunately I no longer have the full backtrace - my bad.
I rebuilt the containers with the changes you committed and my problems seem to have gone now.

I work using docker containers which I've just upgraded to Kamailio v5.8 and the latest Kamcli

In my test environment I have containers built using:
Alpine Linux ( built both Kamailio and Kamcli  from the git repository)
Ubuntu-focal - using  Kamailio  - overnight builds  apt repository  (20.4.6  Python 3.8 )
Ubuntu-jammy - using  Kamailio - overnight builds  apt repository ( 22.04.4     Python 3.10)

I'll let you know if the decode problem re-surfaces.
(The  ed script I gave you only changed   response.decode()  
my original attempt removed all calls to decode() and that  didn't work)
